client/src/app/components/Header/MobileMenu/mobile-menu.css

```diff
@@ -0,0 +1,172 @@
+/*Hamburger styling */
+
+input,
+textarea,
+button,
+select,
+nav,
+a {
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+}
+
+nav {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  display: none;
+  width: 75px;
+  height: 75px;
+  outline: none;
+}
+
+.bar {
+  width: 100%;
+  height: 5px;
+  background-color: #312a31;
+}
+
+nav:hover {
+  cursor: pointer;
+}
+
+#menu {
+  height: 50%;
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+#bar-1.bar-1-animate {
+  animation: bar-1 0.3s;
+  animation-fill-mode: forwards;
+}
+#bar-1.bar-1-animate-rev {
+  animation: bar-1-rev 0.3s;
+  animation-fill-mode: forwards;
+}
+#bar-2.bar-2-animate {
+  animation: bar-2 0.1s;
+  animation-fill-mode: forwards;
+}
+#bar-2.bar-2-animate-rev {
+  opacity: 0;
+  animation: bar-2-rev 0.3s;
+  animation-fill-mode: forwards;
+}
+
+#bar-3.bar-3-animate {
+  animation: bar-3 0.3s;
+  animation-fill-mode: forwards;
+  animation-timing-function: ease-out;
+}
+#bar-3.bar-3-animate-rev {
+  animation: bar-3-rev 0.3s;
+  animation-fill-mode: forwards;
+}
+
+@keyframes bar-1 {
+  0% {
+  }
+
+  50% {
+    transform: translateY(11px);
+  }
+
+  100% {
+    transform: translateY(11px) rotate(45deg);
+    transform-origin: center;
+  }
+}
+
+@keyframes bar-1-rev {
+  0% {
+    transform: translateY(11px) rotate(45deg);
+    transform-origin: center;
+  }
+
+  50% {
+    transform: translateY(11px);
+  }
+
+  100% {
+    transform: translateY(0) rotate(0);
+  }
+}
+
+@keyframes bar-2 {
+  0% {
+  }
+
+  100% {
+    opacity: 0;
+  }
+}
+
+@keyframes bar-2-rev {
+  0% {
+    opacity: 0;
+  }
+  50% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+@keyframes bar-3 {
+  0% {
+  }
+
+  50% {
+    transform: translateY(-11px);
+  }
+
+  100% {
+    transform: translateY(-11px) rotate(-45deg);
+    transform-origin: center;
+  }
+}
+
+@keyframes bar-3-rev {
+  0% {
+    transform: translateY(-11px) rotate(-45deg);
+    transform-origin: center;
+  }
+
+  50% {
+    transform: translateY(-11px);
+  }
+
+  100% {
+    transform: translateY(0) rotate(0);
+    transform-origin: center;
+  }
+}
+
+/* Side menu styling */
+.side-menu {
+  position: absolute;
+  top: 90px;
+  left: 0;
+  width: 300px;
+  background: white;
+  display: none;
+  transition: left 0.5s ease-in-out;
+}
+
+#side-menu-container {
+  padding: 15px;
+}
+
+#side-menu-container > ul {
```

Comment:
It's generally not the best practice to use this kind of selector, or like the one bellow (.class>ul>li). 
Disadvantages: 
- CSS selectors with unspecific tags are slower
- in case you want to change the jsx, or add a wrapper, these selectors won't work anymore

It is best to be consistent and used classes everywhere. Unless ID  really required (eg some 3rd party library requiring an ID to work)